applications/serial_lte_modem/src/main.c

```diff
@@ -54,6 +54,39 @@ void nrf_modem_recoverable_error_handler(uint32_t err)
 	LOG_ERR("Modem library recoverable error: %u", err);
 }
 
+static int ext_xtal_control(bool xtal_on)
+{
+	int err = 0;
```

Comment:
nitpicking...
you could use `IS_ENABLED` instead of ifdef, then you don't need to initialize err.
Something like
```
int err;

if (!IS_ENABLED(CONFIG_SLM_EXTERNAL_XTAL)) {
  return 0;
}
```

Reply:
Please allow me to keep as it is to satisfy Coverity code check.